modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js

```diff
@@ -959,13 +943,7 @@ class Form extends Component {
 			if (saved || published) {
 				shareURLButton.classList.remove('hide');
 			}
-
-			if (defaultLanguageId === editingLanguageId) {
-				this.showAddButton();
-			}
-			else {
-				this.hideAddButton();
-			}
```

Comment:
@carolmariaabb, is there any other place which the logic of display/hide the add button is still necessary? If there is no more need, then we should also remove the methods above and adjust the code to always display the add button.

Reply:
Hi, @natocesarrego,

It is necessary when you switch to the Rules tab and then click on add new rule.

Reply:
Nice job!